### DISCUSSION TITLE
Custom CSS for Remote Theme

### DISCUSSION THREAD

**Discussion by ghost** [community/community#21828] [Other Feature Feedback, Questions, & Ideas]
<p>My <a href="https://github.com/KryptonIon/ItemModifier/tree/master/docs" rel="nofollow noopener">repository</a> has an error whenever building “Your SCSS file assets/css/style.scss has an error on line 1: File to import not found or unreadable: .”</p>
<p>My SCSS looks like this</p>
<pre><code>---
---

@import "";

.inline_header
{
    display: inline;
    margin: 0px;
}
</code></pre>
<p>Thanks in advance!</p>

**Comment by logankilpatrick** [comment]
<p>I think the <code>""</code> is just a place holder. You need to do something like:</p>
<pre><code>@import "{{ site.theme }}";
</code></pre>
<p>or </p>
<pre><code>@import "minima-{{ site.minima.skin | default: 'classic' }}";
</code></pre>
<p>though the first is the most common I have seen. </p>

**Comment by ghost** [comment]
<p>I’ve tried that but it didn’t seem to work, there were a few commits that I removed </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6ca2df0420764332b350dc31c768430b8ad3144a.png" data-download-href="/uploads/short-url/fv2w20JUL1iduU1e3BgeA8nHkN4.png?dl=1" title="chrome_4o1YaITwSF.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/47744493/181067043-5812b03d-24d6-4dd6-ba54-3b03244ae56a.png" alt="chrome_4o1YaITwSF.png" data-base62-sha1="fv2w20JUL1iduU1e3BgeA8nHkN4" width="690" height="17" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6ca2df0420764332b350dc31c768430b8ad3144a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">chrome_4o1YaITwSF.png</span><span class="informations">720×18 2.03 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey! So the error you are getting is saying that it is looking for a file called “just-the-docs.scss” and that that file does not exist, which it doesn’t in your repo. Per here: <a href="https://github.com/KryptonIon/ItemModifier/search?q=just-the-docs&amp;unscoped_q=just-the-docs" rel="nofollow noopener">https://github.com/KryptonIon/ItemModifier/search?q=just-the-docs&amp;unscoped_q=just-the-docs</a> you seem to be setting the theme to “just-the-docs”.</p>
<p>As previously mentioned, you need to choose a theme from the list found here: <a href="https://pages.github.com/themes/" rel="nofollow noopener">https://pages.github.com/themes/</a> thus I do not think that what you are trying to do will work with GitHub pages. You need one of those themes. </p>
